### ISSUE TITLE
Security group rules don't obey custom VPC CIDR option

### ISSUE THREAD

**Issue by bladealslayer** [convox/rack#240]
When a rack is installed using CLI with a custom VPC CIDR and subnet CIDRs, the instance security group rules still contain rules only for `10.0.0.0/8`, regardless of chosen VPC CIDR. As a result, installation hangs at `Waiting for load balancer...` stage, since ELB health checks to instances fail and the kernel service has it's task restarted perpetually.

To reproduce:

```
convox install --vpc-cidr 172.16.0.0/16 --subnet0-cidr 172.16.0.0/24 --subnet1-cidr 172.16.1.0/24 --subnet2-cidr 172.16.2.0/24
```

**Comment by cleblanc87** [comment]
A cursory search indicates this issue is actually present in a few places:

https://github.com/convox/rack/search?utf8=%E2%9C%93&q=%2210.0.0.0%2F16%22&type=Code

**Comment by nzoschke** [comment]
Testing this now with `$ convox install --vpc-cidr 10.101.0.0/16 --subnet0-cidr 10.101.0.0/24 --subnet1-cidr 10.101.1.0/24 --subnet2-cidr 10.101.2.0/24 --stack-name test`

It's not yet clear to me where `10.0.0.0/8` would come from on a fresh install.

**Comment by nzoschke** [comment]
On my newly installing rack, I am not seeing `10.0.0.0/8`, and am seeing `10.101.0.0/16`

<img width="1283" alt="screen shot 2016-05-26 at 10 08 42 am" src="https://cloud.githubusercontent.com/assets/147410/15583238/dd5b272a-2329-11e6-8be8-2e361b9888cd.png">

**Comment by nzoschke** [comment]
My install worked:

```
$ convox install --vpc-cidr 10.101.0.0/16 --subnet0-cidr 10.101.0.0/24 --subnet1-cidr 10.101.1.0/24 --subnet2-cidr 10.101.2.0/24 --stack-name test


     ___    ___     ___   __  __    ___   __  _
    / ___\ / __ \ /  _  \/\ \/\ \  / __ \/\ \/ \
   /\ \__//\ \_\ \/\ \/\ \ \ \_/ |/\ \_\ \/>  </
   \ \____\ \____/\ \_\ \_\ \___/ \ \____//\_/\_\
    \/____/\/___/  \/_/\/_/\/__/   \/___/ \//\/_/


Email Address (optional, to receive project updates): noah@convox.com
This installer needs AWS credentials to install/uninstall the Convox platform into
your AWS account. These credentials will only be used to communicate between this
installer running on your computer and the AWS API.

We recommend that you create a new set of credentials exclusively for this
install/uninstall process and then delete them once the installer has completed.

To generate a new set of AWS credentials go to:
https://docs.convox.com/creating-an-iam-user
AWS Access Key ID: 
AWS Secret Access Key: 

Installing Convox (20160526133520)...
...
Created VPC Internet Gateway: igw-993e41fd
Created VPC: vpc-eefc2889
...
Created VPC Subnet: subnet-c792eded
Created VPC Subnet: subnet-d8087280
Created VPC Subnet: subnet-4c09273a
...
Waiting for load balancer...
Logging in...
Success, try `convox apps`
```

**Comment by nzoschke** [comment]
I'll test with the `172.16.0.0` blocks too.

**Comment by nzoschke** [comment]
This is related to https://github.com/convox/rack/issues/637, which I'm looking at too.

**Comment by nzoschke** [comment]
Turns out this was fixed in 7913b028dc231af5037a855c7a936ad498263b83. Closing this and turning attention to #637.
